.github/workflows/build.yml

```
@@ -531,7 +531,7 @@ jobs:
             mv ${{ matrix.result_dir }}/${{ matrix.artifact_name }}/{,ORT_}LICENSE
             mv ${{ matrix.result_dir }}/${{ matrix.artifact_name }}/{,ORT_}README.md
             mv ${{ matrix.result_dir }}/${{ matrix.artifact_name }}/{,Ort}ThirdPartyNotices.txt
-            mv ./VOICEVOX_LICENSE.md ${{ matrix.result_dir }}/${{ matrix.artifact_name }}/LICENSE.md
+            mv ./VOICEVOX_ORT_TERMS.md ${{ matrix.result_dir }}/${{ matrix.artifact_name }}/
```

単に"TERMS.md"として入れるのもありかもととはちょっと思いました。この下のthird-party-notices.htmlに合わせる意味でも。

```suggestion
            mv ./VOICEVOX_ORT_TERMS.md ${{ matrix.result_dir }}/${{ matrix.artifact_name }}/TERMS.md
```

---

たしかに、TERMS.mdが良さそうな気がしました！
仮にREADMEがあった場合も`voicevox_onnxruntime`の`README.md`でしょうし。

あと`ORT`というのは何なのか伝わらないと思うので、onnxruntimeにしておくかあるいは別の一般的な名称をつけるとかの方が良いかも。